foreman/data_refinery_foreman/surveyor/test_surveyor.py

```diff
@@ -26,7 +26,7 @@ def test_calls_survey(self, survey_method):
         job.save()
         surveyor.run_job(job)
 
-        survey_method.assert_called_with(job)
+        survey_method.assert_called_with()
```

Comment:
This should just be `assert_called`. The `survey_method` Mock is passed in by the `@patch` decorator, which is mocking `surveyor.ArrayExpressSurveyor`'s `survey` method.

The reason I no longer am passing job to this method is that the `ArrayExpressSurveyor` object has the job as an instance variable, so it doesn't need to have it passed in.